tests/Doctrine/Tests/Annotations/Type/Constant/ConstantFloatTypeTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Annotations\Type\Constant;
+
+use Doctrine\Annotations\Type\Constant\ConstantFloatType;
+use PHPUnit\Framework\TestCase;
+use stdClass;
+
+final class ConstantFloatTypeTest extends TestCase
+{
+    public function testGetValue() : void
+    {
+        self::assertEquals(1.23, (new ConstantFloatType(1.23))->getValue(), '', 1e-5);
+        self::assertEquals(1.1111111111, (new ConstantFloatType(1.1111111111))->getValue(), '', 1e-10);
+    }
+
+    public function testDescribe() : void
+    {
+        self::assertSame('1.230000', (new ConstantFloatType(1.23))->describe());
+        self::assertSame('1.111111', (new ConstantFloatType(1.1111111111))->describe());
+    }
+
+    public function testValidate() : void
```

Comment:
There needs to be testcase proving the need for epsilon

Reply:
It's already covered by 2nd assert.

Reply:
That doesn't cover it
```php
>>> 1.1111111111 === 1.1111111111000000000000000000000000001
=> true
```

Reply:
1.1111111111000009 will be better choice.